ccm_web/server/src/canvas/canvas.service.ts

```diff
@@ -32,13 +39,14 @@ const requestorOptions: GotOptions = {
     ],
     afterResponse: [
       (response, retryWithMergedOptions) => {
-        logger.debug(`afterResponse — "x-rate-limit-remaining": "${response.headers['x-rate-limit-remaining'] || ''}"; "x-request-cost": "${response.headers['x-request-cost'] || ''}"`)
+        const headers = response.headers as IncomingRateLimitedCanvasHttpHeaders
+        logger.debug(`afterResponse — "x-rate-limit-remaining": "${headers['x-rate-limit-remaining'] ?? ''}"; "x-request-cost": "${headers['x-request-cost'] ?? ''}"`)
```

Comment:
`?? ''` syntax is effective here, though I tend to use the `String(...)` cast as then you still see the string `undefined`. Minor difference though.